File: Python/stage2/S2_ARR_01.py
# S2_ARR_01 — Clean numbers
def clean_numbers(values):
   result = []
   for v in values:
       try:
           n = float(v)
           result.append(n)
       except ValueError:
           pass
   return result
data = [" 1 ", "x", "2", " 3.5 ", "abc"]
print(clean_numbers(data))

File: js/stage2/S2_ARR_01.js
// S2_ARR_01 - Clean numbers
function cleanNumbers(arr) {
 return arr
   .map(x => +x)
   .filter(x => !Number.isNaN(x));
}
const input = [" 1 ", "x", "2"];
const output = cleanNumbers(input);
console.log("Input:", input);
console.log("Output:", output);
